src/Pages/Donation/index.tsx

```diff
@@ -3,17 +3,14 @@ import { DonationForm } from "./style";
 import { InputGroup } from "./style";
 import { InputLabel } from "./style";
 import { InputField } from "./style";
-import { SubmitButton } from "./style";
 import { PageBackground } from "./style";
+import axios from "axios";
 
-interface DonationFormData {
-  name: string;
-  email: string;
-  amount: number;
-}
+const APP_ID = "33221729445f634e544edbc48c712233";
```

Comment:
app id should not be stored as a constant here. It should be used from .ev files. It's a secret that should not be pushed to git repo